src/Sylius/Bundle/OrderBundle/DependencyInjection/Configuration.php

```diff
@@ -144,6 +147,19 @@ private function addResourcesSection(ArrayNodeDefinition $node)
                                 ->end()
                             ->end()
                         ->end()
+                        ->arrayNode('order_item_unit')
+                            ->addDefaultsIfNotSet()
+                            ->children()
+                                ->arrayNode('classes')
+                                    ->addDefaultsIfNotSet()
+                                    ->children()
+                                        ->scalarNode('model')->defaultValue(OrderItemUnit::class)->cannotBeEmpty()->end()
+                                        ->scalarNode('interface')->defaultValue(OrderItemUnitInterface::class)->cannotBeEmpty()->end()
+                                        ->scalarNode('factory')->defaultValue(OrderItemUnitFactory::class)->end()
```

Comment:
I suppose we should have `->scalarNode('repository')->cannotBeEmpty()->end()` also.